pkg/commands/git_commands/commit.go

```diff
@@ -74,6 +74,12 @@ func (self *CommitCommands) GetCommitMessage(commitSha string) (string, error) {
 	return strings.TrimSpace(message), err
 }
 
+func (self *CommitCommands) GetCommitDiff(commitSha string) (string, error) {
+	cmdStr := "git show " + commitSha
```

Comment:
I'd add the --no-color flag here given that we don't need colour when copying to clipboard